tiledb/sm/buffer/buffer.h

```diff
@@ -276,6 +303,148 @@ class Buffer {
   Status ensure_alloced_size(uint64_t nbytes);
 };
 
+// =======================================================
+//      ConstBuffer
+// =======================================================
```

Comment:
Add newline?

Reply:
I've got the section headers attached to the doc-comment of the class that's being defined.